tests/acceptance/features/bootstrap/WebDavPropertiesContext.php

```diff
@@ -616,6 +615,17 @@ public function valueOfItemOfPathShouldBe(string $user, string $xpath, string $p
 	 * @return void
 	 * @throws Exception
 	 */
+	public function theValueOfTheItemInTheResponseAboutUserShouldBe(string $xpath, ?string $user, string $expectedValue):void {
```

Comment:
also remove null from code docs
```suggestion
	public function theValueOfTheItemInTheResponseAboutUserShouldBe(string $xpath, string $user, string $expectedValue):void {
```